### DISCUSSION TITLE
Topics of interest

### DISCUSSION THREAD

**Discussion by lewisfogden** [actuarialopensource/website#31] [Team Posts]
Thought I'd start a list of topics of interest

- Projections / cashflow modelling (e.g. Life Office Projections,  Customer Portfolio Projections)
- Machine Learning / AI  -  this is more a toolset than an actuarial topic.
- GI Pricing (Risk / Commercial)*
- Experience analysis (assumption setting)
- Capital / Risk modelling (e.g. SII / stochastic simulations / proxy models)

*Is this possible to discuss in general terms without being commercially sensitive?

What open source libraries / toolkits exist for each?

**Comment by fumitoh** [comment]
I took your list, broaden it to cover all the potential topics in this community::

- Data Science
  - Machine Learning / AI
- Life Insurance
  - Projections / cashflow modeling..
  - ALM, VA hedging
  - Economic scenario generators
  - Expense analysis 
- Pensions (Defined Benefit
  - scheme valuations
- P&C
  - GI Pricing..
  - Reserving
  - (There must be something..)
- Risk Management
  - Capital / Risk modeling ..
  - Market Risk, Credit Risk modeling
  - Model Risk Management
  - Model Governance
- Financial reporting standards:
  - Solvency 2
  - IFRS17
- Actuarial modernization
  - Process automation

For ALM (asset models) & Market risk, there may be something useful in [this list](https://github.com/wilsonfreitas/awesome-quant).
